Header header
float64 feature_extraction #(current time - t1)
float64 stereo_matching #(current time - t2)
float64 compute_3D_points #(current time - t3)
float64 need_new_keyframe #(current time - t4)
float64 total #(current time - t0)